examples/flakes/php/my-php-flake/flake.nix

```diff
@@ -20,7 +20,7 @@
 
           # If you only want to export a single package, you can name it default which allows
           # installation without using url fragment (.e.g. "path:my-flake")
-          default = pkgs.php.withExtensions ({ enabled, all }: enabled ++ (with all; [ ds memcached ]));
+          # default = pkgs.php.withExtensions ({ enabled, all }: enabled ++ (with all; [ ds memcached ]));
```

Comment:
could we create another example so we have:
1. one example with default export
2. one example without a default export

?

Reply:
I can modify the php-with-extensions to have a default export, and push it to this pr